platform-includes/getting-started-config/javascript.solidstart.mdx

```diff
@@ -0,0 +1,102 @@
+To set up the Sentry SDK initialize it in the client and server.
+
+### Client-side Setup
+
+Initialize the Sentry SDK in your `entry-client.tsx` file.
+
+If you're using Solid Router, add the `solidRouterBrowserTracingIntegration` to collect meaningful performance data about the health of your page loads and associated requests.
+
+<SignInNote />
+
+```jsx {filename:entry-client.tsx}
+import * as Sentry from '@sentry/solidstart';
+import { solidRouterBrowserTracingIntegration } from '@sentry/solidstart/solidrouter';
+import { mount, StartClient } from '@solidjs/start/client';
+
+Sentry.init({
+  dsn: '__PUBLIC_DSN__',
+  integrations: [solidRouterBrowserTracingIntegration()],
+  tracesSampleRate: 1.0, //  Capture 100% of the transactions
+});
+
+mount(() => <StartClient />, document.getElementById('app'));
+```
+
+### Server-side Setup
+
+Create an instrument file `instrument.server.mjs` and initialize the Sentry SDK.
```

Comment:
M: Specify, where this file should be located. If it isn't root, you can also add the path to the filename in the code snippet.

Reply:
Updated